telco-core/configuration/reference-crs-kube-compare/required/networking/multinetworkpolicy/multiNetworkPolicyDenyAll.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: k8s.cni.cncf.io/v1beta1
```

Comment:
multinetwork policy resources should only be required if the customer enabled them in network.operator

Reply:
Agreed, it is an optional component. I've noted in metadata.yaml that this should be optional.